src/_guides/openapi/specification/v3.1/understanding-structure/http-requests.md

```diff
@@ -0,0 +1,105 @@
+---
+title: HTTP Requests
+authors: phil
+excerpt: Define HTTP requests in your OpenAPI to help users know the rules on what to send and how.
+date: 2024-07-09
+---
+
+Any API handling use-cases more advanced that purely fetching data will need to define a HTTP request body. `POST`, `PATCH`, `PUT`, all allow a HTTP client to send a body: often JSON or XML. This allows more information can be sent than just query parameters, which have limits. 
```

Comment:
```suggestion
Any API handling use-cases more advanced that purely fetching data will need to define a HTTP request body. `POST`, `PATCH`, `PUT`, all allow a HTTP client to send a body: often JSON or XML. This allows for more information to be sent rather than just query parameters, which have limits. 
```